packages/server/lib/plugins/dev-server.js

```diff
@@ -32,8 +33,7 @@ const API = {
 
   start ({ specs, config }) {
     if (!plugins.has('dev-server:start')) {
-      // TODO: add link to the documentation in the error message
-      throw new Error('It is required to register dev-server plugin that implements `dev-server:start` event for component testing.')
+      return errors.throw('CT_NO_DEV_START_EVENT', config.pluginsFile)
```

Comment:
Is the second arg (`config.pluginsFile`) needed? Seems it is not used.

Reply:
Good catch, my plan was to use it in the error to offer a clickable target for the user.

`Here is the path to your pluginsFile: /Users/dev/path`